src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin square {
+  width: $ceil-width;
+  height: $ceil-height;
+  border: 1px solid black;
```

Comment:
`square` could be without a border too, so it's better to remove the border from mixin